src/Exception/MethodNotAllowedException.php

```diff
@@ -2,8 +2,8 @@
 
 namespace Opstalent\CrudBundle\Exception;
 
-use Throwable;
 use LogicException;
+use Throwable;
```

Comment:
I assume this is because of linter use?

Reply:
this is for @param Throwable|null $previous